tests/storage/datavolume.go

```diff
@@ -76,10 +76,30 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 		}
 	})
 
-	Context("[rook-ceph]PVC expansion", func() {
+	Context("[block-storage]PVC expansion", func() {
```

Comment:
right. of course I can add another category StorageExpansion. in u/s it will be only for ceph. for d/s? hpp? nfs?
Though what is the limit, I can say the same for WFFC storage class also.. do we want that too?

Reply:
Neither hpp or NFS can do volume expansion. So for now it would just be ceph. But others might join the party soon (topoLVM comes to mind). WFFC is a good question, it is certainly a capability (or caveat really) of the storage. And we have some tests specifically for it, to ensure it works properly. Think about the doppelganger pod we use to trigger the right binding of PVCs. The question should we make it a separate config option, I am not sure, it would really only be relevant for the specific WFFC tests.